scss/dialogs/_view_profile.scss

```diff
@@ -58,5 +58,17 @@ div.profile-info-container {
     &.clickable:hover {
       cursor: pointer;
     }
+
+    & > i.material-icon {
```

Comment:
we might have other material icons, so maybe better add the bulk of this calls to some class maybe named `.material-svg-icon` and then also add a class `.material-icon-schedule`? and do this in the scss file for icons. then we can also reuse the class elsewhere and would just overwrite things like size in viewProfile if even necessary, Basing it on `em` is a good choice so probably such overwrites are not necessary.

Reply:
Agreed.

Reply:
@Simon-Laux How is it now?